tests/test-timber.php

```diff
@@ -772,16 +717,6 @@ public function testGettingEmptyArray()
         $this->assertEquals([], $collection->to_array());
     }
 
-    public function testFromFalse()
-    {
-        // We don't actually test this directly in TestTimberPostGetter::testGettingWithFalse();
-        // that test directly instantiates a collection.
-        $this->markTestSkipped('@todo what should this be?');
```

Comment:
With the newest version, when using `false` with `Timber::get_posts()`, the default query will be used. This test would test a deprecated functionality then. That’s why I removed it.